docs/newbs_getting_started.md

```diff
@@ -1,99 +1,97 @@
+
+You can also download and install a dedicated text editor like [Sublime Text](https://www.sublimetext.com/) or [VS Code](https://code.visualstudio.com/). This is probably the best way to go regardless of platform, as these programs are specifically made for editing code.
 
 ?> Not sure which text editor to use? Laurence Bradford wrote [a great introduction](https://learntocodewith.me/programming/basics/text-editors/) to the subject.
 
 ### QMK Toolbox
 
-QMK Toolbox is an optional graphical Windows and macOS program that allows you to both program and debug your custom keyboard. You will likely prefer it to easily flash your keyboard and receive the debugging messages that your keyboard will print.
-
-Download the files from the links below:
-
-For Windows: "qmk_toolbox.exe" or "qmk_toolbox_install.exe" (with installer)
+QMK Toolbox is an optional graphical program for Windows and macOS that allows you to both program and debug your custom keyboard. You will likely find it invaluable for easily flashing your keyboard and viewing debug messages that it prints.
 
-For Mac: "QMK.Toolbox.app.zip" or "QMK.Toolbox.pkg" (with installer)
+[Download the latest release here.](https://github.com/qmk/qmk_toolbox/releases/latest)
 
-* [Newest Release](https://github.com/qmk/qmk_toolbox/releases/latest)
-* [Source Code](https://github.com/qmk/qmk_toolbox/)
+* For Windows: `qmk_toolbox.exe` (portable) or `qmk_toolbox_install.exe` (installer)
+* For macOS: `QMK.Toolbox.app.zip` (portable) or `QMK.Toolbox.pkg` (installer)
 
-## Environment Setup
+## Set Up Your Environment
 
-We've tried to make QMK as easy to setup as possible. You only have to prepare your Linux or Unix environment and let QMK install the rest.
+We've tried to make QMK as easy to set up as possible. You only have to prepare your Linux or Unix environment, then let QMK install the rest.
 
-?> If you haven't worked with the Linux/Unix command line before there are a few basic concepts and commands you should learn. These resources will teach you enough to work with QMK:<br>
+?> If you haven't worked with the Linux/Unix command line before, there are a few basic concepts and commands you should learn. These resources will teach you enough to be able to work with QMK:<br>
 [Must Know Linux Commands](https://www.guru99.com/must-know-linux-commands.html)<br>
 [Some Basic Unix Commands](https://www.tjhsst.edu/~dhyatt/superap/unixcmd.html)
 
 ### Windows
 
-You will need to install msys2 and git.
+You will need to install MSYS2 and Git.
 
-* Follow the installation instructions on the msys2 homepage: http://www.msys2.org
-* Close any open msys2 terminals, and open a new terminal
-* Install git by running this command: `pacman -S git`
+* Follow the installation instructions on the [MSYS2 homepage](http://www.msys2.org).
+* Close any open MSYS2 terminals and open a new terminal.
```

Comment:
```suggestion
* Close any open MSYS2 terminals and open a new "MINGW64" terminal.
```

Specifically, MSYS installs three options, and we should always be using MINGW64.

Reply:
@drashna 

> Specifically, MSYS installs three options, and we should always be using MINGW64.

Why is that? I've been using the MSYS terminal for a few weeks, without issue.

Reply:
It's the binaries that it uses.  The MINGW uses native windows binary for the architecture in question.  MINGW64 uses 64 bit Windows binaries, which is better, since it's less emulation

Reply:
Also, it's what we instruct elsewhere

Reply:
How about:
```suggestion
* Close any open MSYS2 terminals and open a new MSYS2 MinGW 64-bit terminal.
```
I believe that's the exact wording that MSYS2 itself uses.

Reply:
Ah, found the doc I was looking for: https://github.com/msys2/msys2/wiki/MSYS2-introduction#subsystems

> MSYS2 consists of three subsystems and their corresponding package repositories, msys2, mingw32, and mingw64.
> 
> The mingw subsystems provide native Windows programs and are the main focus of the project. These programs are built to co-operate well with other Windows programs, independently of the other subsystems. This part builds on the MinGW-w64 project.
> 
> The msys2 subsystem provides an emulated mostly-POSIX-compliant environment for building software, package management, and shell scripting. These programs live in a virtual single-root filesystem (the root is the MSYS2 installation directory). Some effort is made to have the programs work well with native Windows programs, but it's not seamless. This part builds on the Cygwin project.
> 
> Each of the subsystems provides its own native (i.e. target=host) compiler toolchain, in msys2-devel, mingw-w64-i686-toolchain, and mingw-w64-x86_64-toolchain. There are also cross compiler toolchains with host={i686,x86_64}-pc-msys and target={i686,x86_64}-w64-mingw32 in mingw-w64-cross-toolchain, but these are of limited use because there are no library packages for them.
>